### ISSUE TITLE
Tech Report: Overall good CWV vs Origins mismatch

### ISSUE THREAD

**Issue by konfirmed** [HTTPArchive/httparchive.org#991]
<img width="1323" alt="Image - overal was meant to be overall 👎🏽" src="https://github.com/user-attachments/assets/a5df32fe-0acb-44bf-8dd1-5afa0edc6069" />

I have a question about this, Is this meant to be so? if yes, I am a bit confused by the data. Does the filter work for the overall good CWV?

**Comment by tunetheweb** [comment]
Is overall good a percentage? If so, it should be made clearer in the UI.

CC: @sarahfossheim

**Comment by max-ostapenko** [comment]
'Overall Good CWV' is a percentage calculated with the data from the endpoint `good_number / tested`. Needs formatting.

**Comment by sarahfossheim** [comment]
Yes, should be a percentage 🤔 will fix the formatting in the open PR.
